Nos/Controller/PersistenceController.swift

```diff
@@ -223,4 +223,8 @@ class PersistenceController {
             return statistics.sorted(by: { $0.key < $1.key })
         }
     }
+
+    func sqliteURL() -> URL? {
```

Comment:
This might be better described with a var instead of a func, unless the computation is expensive, in that case, it's ok to be a func

Reply:
done!